_data/squad_call.yml

```diff
@@ -0,0 +1,58 @@
+# Squad Call
```

Comment:
These and the other recurring calls are others that could be pulled from the groups.io calendar. Or, it might be easier to embed a Google Calendar that is subscribed to those calendars.

Reply:
Yes, both might work. Can you provide the API to be pulled doesn't seem to find any on [OMP Zowe Calendar](https://lists.openmainframeproject.org/g/zowe-dev/calendar), or the Embed code.

Reply:
Here's an ical feed... https://lists.openmainframeproject.org/ics/5878867/8526030431515617925/feed.ics

Reply:
Thank you, but I'm not sure if it provides the information in the way we are looking. I'm looking if we could divide the schedules "Squad wise".

I looked upon the response, it didn't seem to provide events which are going to happen in future. Instead of past months from 2020, and some of 2018, 2019 as well. It also seem to provide some extra meetings on the specific day as compared to this [calendar](https://lists.openmainframeproject.org/g/zowe-dev/calendar)

It will be really feasible, if we could embed a Groups.io calendar or a Google Calendar that is subscribed to it directly. Instead of pulling it from the ical feed.

Reply:
Thank you, but I'm not sure how we will be able to divide the response and meetings schedules "Squad wise".

I looked upon the response, it didn't seem to provide events which are going to happen in future. Instead of past months from 2020, and some of 2018, 2019 as well. It also seem to provide some extra meetings on the specific day as compared to this [calendar](https://lists.openmainframeproject.org/g/zowe-dev/calendar)

It will be really feasible, if we could embed a Groups.io calendar or a Google Calendar that is subscribed to it directly. Instead of pulling it from the ical feed.

Reply:
So it's all one calendar managed in groups.io, so you can't really subdivide it up very well.

Let me more on at least getting you an embeddable Google Calendar, since that should make this easier.

Reply:
Yes, this blog on "[How to Sync Google Calendar with groups.io](https://groups.io/g/GroupManagersForum/wiki/5138)" - might help

Reply:
Here you go....

```html
<iframe src="https://calendar.google.com/calendar/embed?src=c55ndlc8frlvhau31r13fggmvfg4qmeu%40import.calendar.google.com&ctz=America%2FNew_York" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
```

Reply:
Thanks incorporated it, I think this PR is ready to be merged now!